Userland/Libraries/LibWeb/Layout/BlockContainer.h

```diff
@@ -22,8 +26,13 @@ class BlockContainer : public Box {
 
     Painting::PaintableWithLines const* paintable_with_lines() const;
 
+    void store_layout_inside_run_info(LayoutState* layoutState, AvailableSpace const availableSpace);
```

Comment:
Capitalization. (This one, you can just remove the variable names because they're the same as the type names.

```suggestion
    void store_layout_inside_run_info(LayoutState*, AvailableSpace const);
```